packages/dmn-js-drd/test/spec/features/context-pad/ContextPadProviderSpec.js

```diff
@@ -166,6 +166,60 @@ describe('features - context-pad', function() {
       })
     );
 
+    describe('multi remove', function() {
```

Comment:
Code style convention which should but is not part of eslint ruleset. One empty line after describe, two empty lines between test cases.

```suggestion
    describe('multi remove', function() {

```

Reply:
do you have a link to this style convention? I was looking for it but couldn't find it yet

Reply:
This is how we've been writing the code for years (e.g. https://github.com/bpmn-io/bpmn-js/blob/develop/test/spec/features/drilldown/DrilldownSpec.js), but it's unfortunately undocumented.

Reply:
done